addons-cluster/redis/templates/_helpers.tpl

```diff
@@ -243,13 +262,16 @@ metadata:
   name: {{ include "kblib.clusterName" . }}
   namespace: {{ .Release.Namespace }}
   labels: {{ include "kblib.clusterLabels" . | nindent 4 }}
-  {{- if and .Values.hostNetworkEnabled (eq .Values.mode "cluster") }}
   annotations:
+    apps.kubeblocks.io/mode: {{ .Values.mode }}
+  {{- if and .Values.hostNetworkEnabled (eq .Values.mode "cluster") }}
     kubeblocks.io/host-network: "shard"
   {{- else if .Values.hostNetworkEnabled }}
-  annotations:
     kubeblocks.io/host-network: "redis,redis-sentinel"
   {{- end }}
+  {{- if and .Values.podAntiAffinityEnabled (eq .Values.mode "cluster") }}
+    apps.kubeblocks.io/shard-pod-anti-affinity: "shard"
```

Comment:
ditto

Reply:
Needed. 
The primary and secondary replicas of each sharding implement by this annotation

Reply:
I didn't get it. The annotation seems not to appear anywhere else in the code. If these annotations are for use by other external systems, I suggest adding a new field in values.yaml (e.g., annotations) and set them when needed. So do the `apps.kubeblocks.io/mode` annotation.